specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/examples/SuccessfulAnalyzeConversationTextJobsTaskSubmitRequest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "parameters": {
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "api-version": "2022-04-01-preview",
+    "Endpoint": "{Endpoint}",
+    "jobId": "{Job ID}",
+    "body": {
+      "displayName": "Analyze conversations from xxx",
```

Comment:
Please rephrase. This example text will be use in the REST console experience. 
Use something like `"Contoso meeting"`.

Reply:
updated